### DISCUSSION TITLE
Device Groups not working as expected

### DISCUSSION THREAD

**Discussion by bkbartk** [arendst/Tasmota#14533] [Q&A] [answered]
Hello,

I have 2 device groups configured, 
1 contains 2 Sonoff MiniR2 devices,
this one is working,
the other 1 contains 2 Sonoff T1 1CH devices.
for this one device grouping is not working.
`{"DevGroupStatus":{"Index":0,"GroupName":"Attic","MessageSeq":2,"MemberCount":0,"Members":[]}}`
DevGroupStatus doesn't show any other devices.
And I don't know where I went wrong,

on all devices I run "Tasmota 10.1.0"
and I would think this might be a network issue, but if that would be the case the first group wouldn't work either.

Is there a way to debug the issue for me?

**Comment by sfromis** [comment, marked as answer]
You may not have set up the grouping correctly, with all details. Better go through it step by step. 
https://tasmota.github.io/docs/Device-Groups/

**Reply by bkbartk** [reply]
thanks for the reply,
it's now an hour later, I didn't change anything and suddenly it started to work.
So I don't know how or what, but it works,
